### ISSUE TITLE
Clustergrams do not show dendrograms

### ISSUE THREAD

**Issue by danlovesbiophysics** [plotly/dash-bio#527]
**Describe the bug**
Clustergram is missing both dendrograms

**To Reproduce**

```
import dash_bio as dashbio
import pandas as pd

df = pd.read_csv(
    'https://raw.githubusercontent.com/plotly/dash-bio-docs-files' +
    '/master/clustergram_mtcars.tsv',
    sep='	', skiprows=4
).set_index('model')

data = df.values

fig = dashbio.Clustergram(
  data=data,
  color_threshold={'row': 150, 'col': 700},
  column_labels=list(df.columns.values),
  row_labels=list(df.index),
  hidden_labels=['row'],
  width=600
)

fig.show()
```
Note, this code throws a warning that could be relevant: 
```
python3.8/site-packages/plotly/tools.py:461: DeprecationWarning:
plotly.tools.make_subplots is deprecated, please use plotly.subplots.make_subplots instead
```

**Expected behavior**
Should show the heatmap + side/top dendrograms

**Screenshots**

<img width="574" alt="Screen Shot 2020-11-05 at 2 54 16 PM" src="https://user-images.githubusercontent.com/67762159/98300822-0fb4ab80-1f77-11eb-9a1b-e7000a3d9e0a.png">

**Python version: [e.g., 3.7.2]**

3.8

**Python environment (all installed packages in your current environment):**

dash_bio 0.4.8
pandas 1.1.4
plotly 4.12.0

**Additional context**
The missing dendrograms also affect the "Clustergram Examples and Reference" page at https://dash.plotly.com/dash-bio/clustergram

**Comment by alexcjohnson** [comment]
Duplicate of #507 and fixed by #512 but it's sat unreleased for quite a while. We were hoping to get a few more of the open PRs merged before making a new release (in particular #516) but it's been taking a while - @HammadTheOne @Marc-Andre-Rivet maybe we should release what's already merged, then do another once IGV is in?

**Comment by HammadTheOne** [comment]
![image](https://user-images.githubusercontent.com/30986043/98583191-f91d9580-2291-11eb-86a7-eaaf13b7779d.png)

Thanks for pointing this out! As Alex mentioned, this bug is fixed in the `0.5.0` release of `dash-bio`. Closing this issue.
